news/management/commands/process_donations_queue.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import print_function, unicode_literals
+
+import json
+import sys
+from time import time
+
+from django.conf import settings
+from django.core.management import BaseCommand, CommandError
+
+import boto3
+import requests
+from django_statsd.clients import statsd
+
+from news.tasks import process_donation
+
+
+class Command(BaseCommand):
+    snitch_delay = 300  # 5 min
+    snitch_last_timestamp = 0
+    snitch_id = settings.DONATE_SNITCH_ID
+
+    def snitch(self):
+        if not self.snitch_id:
+            return
+
+        time_since = int(time() - self.snitch_last_timestamp)
+        if time_since > self.snitch_delay:
+            requests.post('https://nosnch.in/{}'.format(self.snitch_id))
```

Comment:
Just playing nice with DMS. We don't need to ping every time as they'll be coming in quite frequently during the height of the campaign. And yes indeed, it would be a great addition to babis! Also thanks for the link to babis, I had not heard of it. A very useful utility!